enterprise/course_catalog_api.py

```diff
@@ -244,6 +244,10 @@ def get_common_course_modes(self, course_run_ids):
 
         return available_course_modes
 
+    def course_in_catalog(self, catalog_id, course_run_id):
```

Comment:
not something to address in the scope of this PR, but i'm wondering if we should align on whether or not the enterprise api is the only thing within these packages that calls to the catalog api directly, and everything else uses the enterprise api.